pkg/server/status_local_file_retrieval.go

```diff
@@ -59,6 +62,24 @@ func profileLocal(
 			return nil, err
 		}
 		return &serverpb.JSONResponse{Data: buf.Bytes()}, nil
+	case serverpb.ProfileRequest_GOROUTINE:
+		p := pprof.Lookup("goroutine")
+		if p == nil {
+			return nil, status.Error(codes.InvalidArgument, "unable to find goroutine profile")
+		}
+		var debug int
+		buf := bytes.NewBuffer(nil)
+		if req.WithLabels {
+			debug = 1
```

Comment:
nit: why the local `debug` and not just `req.WithLabels` everywhere?

Reply:
My thought was to `s/debug/req.WithLabels/g`. There's just one use of the `debug` local. Easier to read, for me at least, if the indirection is avoided. This is minor, so your call!